client/src/components/pages/admin/Submitted_Id/index.js

```diff
@@ -1,5 +1,382 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Helmet } from 'react-helmet';
+import Checkbox from '@material-ui/core/Checkbox';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import Limitation from '../../../common/limitation';
+import Card from '../../../common/card';
+import Typography from '../../../common/Typography';
+import Button from '../../../common/Button';
+import backGround from '../../../../assets/images/backgroundDash.svg';
+import profileImage from '../../../../assets/images/HomeImg.svg';
+import DashBar from '../../../dashboard/Tabs';
+import './index.css';
 
-const SubmittedId = () => <h1>SubmittedId </h1>;
+const applicantData = async (id) => {
+  const {
+    data: { user },
+  } = await axios.get(`/api/v1/applicants/${id}`);
+  return user;
+};
+
+const getTec = async () => {
+  const { data } = (await axios.get('/api/v1/tasks')).data;
+  return data;
+};
+
+const updatePoints = async (id) => {
+  const Points = await axios.get(`/api/v1/applicants/${id}/points`);
+  return Points;
+};
+
+const SubmittedId = ({ location: { pathname } }) => {
+  const [data, setData] = useState();
+  const [Technical, setTechnical] = useState();
+  const [UserId, setId] = useState('');
+  const [acceptedVal, setAcceptedVal] = useState();
+  const history = useHistory();
+
+  useEffect(() => {
+    if (pathname) {
+      setId(pathname.split('/')[4]);
+    }
+    if (!data && UserId) {
+      updatePoints(UserId).then();
+      applicantData(UserId).then(setData);
+      applicantData(UserId).then(({ accepted }) => setAcceptedVal(accepted));
+    }
+    if (!Technical) getTec().then(setTechnical);
+  }, [data, Technical, pathname, UserId, acceptedVal]);
+
+  const setAccept = async (id, acceptedValue) => {
```

Comment:
@alidah0 You could move this function outside of the component, similar to what you did with getTec and updatePoints, and then the function won't be redeclared every time the component re-renders.